### PR TITLE
feat(addon-commerce): `InputCard` uses `Maskito` instead of legacy `text-mask`

### PR DESCRIPTION
**Pull request by nsbarsukov**
## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [X] Refactoring
- [ ] Build or CI related changes
- [ ] Documentation content changes

## What is the current behavior?
Partially solves #120

## Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

### CODE REVIEW

File: projects/addon-commerce/components/input-card/input-card.component.ts [hunk 1]
```diff
@@ -97,9 +96,6 @@ export class TuiInputCardComponent extends AbstractTuiInputCard<string> {
 
     @tuiPure
     private getFormattedCard(cardNumber: string): string {
-        return cardNumber
-            .split('')
-            .map((char, index) => (index && index % 4 === 0 ? ` ${char}` : char))
-            .join('');
+        return maskitoTransform(cardNumber, this.maskOptions);
```
Comment (nsbarsukov) [inline projects/addon-commerce/components/input-card/input-card.component.ts:99]: oops, mistake

I should change it to
```ts
    private getFormattedCard(cardNumber: string, options: MaskitoOptions): string {
        return maskitoTransform(cardNumber, options);
    }
```

Comment (nsbarsukov) [reply projects/addon-commerce/components/input-card/input-card.component.ts]: fixed

File: projects/addon-commerce/components/input-card/test/input-card.component.spec.ts [hunk 2]
```diff
@@ -156,6 +156,22 @@ describe(`InputCard`, () => {
...
+
+        it(`7`, async () => {
+            await testFormat(`1234567`, `1234 567`);
+        });
+
+        it(`10`, async () => {
+            await testFormat(`1234567890`, `1234 5678 90`);
+        });
+
+        it(`12`, async () => {
```
Comment (splincode) [inline projects/addon-commerce/components/input-card/test/input-card.component.spec.ts:171]: The card number can have 13, 15, 16, 18 and even 19 digits. The most common option is 16 digits.
Do we have test suites for that?

Comment (nsbarsukov) [reply projects/addon-commerce/components/input-card/test/input-card.component.spec.ts:171]: Yep, few lines below this comment)

File: projects/addon-commerce/components/input-card/input-card.component.ts [hunk 3]
```diff
@@ -59,7 +58,7 @@ export class TuiInputCardComponent extends AbstractTuiInputCard<string> {
     }
 
     get formattedCard(): string {
```
Comment (waterplea) [inline projects/addon-commerce/components/input-card/input-card.component.ts:60]: This is only used in template, we can use maskito pipe.

**Review by waterplea — COMMENTED** [review]
What about input card inside `InputCardGroup`?

File: projects/addon-commerce/components/input-card/input-card.component.ts (hunk 3 above)
Comment (nsbarsukov) [reply projects/addon-commerce/components/input-card/input-card.component.ts:60]: You are right, I've forgot about this pipe))

**Comment by nsbarsukov** [comment]
> What about input card inside InputCardGroup?

@waterplea I will do it in next separate PR

File: projects/addon-commerce/components/input-card/input-card.component.ts (hunk 3 above)
Comment (nsbarsukov) [reply projects/addon-commerce/components/input-card/input-card.component.ts]: fixed

**Review by waterplea — APPROVED** [review]

**Review by vladimirpotekhin — APPROVED** [review]

**Pull request merged**